en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+and organization ID. Optionally, you can configure the `geoLocationFilter` parameter to further filter the results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">geoLocationFilter</td>
+<td valign="top"><a href="#geolocationfilter">GeoLocationFilter</a></td>
+<td>
+
+Geo location filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getAPIsUsageByApplications</strong></td>
+<td valign="top">[<a href="#apiusagebyapplication">APIUsageByApplication</a>]</td>
+<td>
+
+Returns a list of each API usage by each application within the specified time range for the given combination of
+tenant ID, environment ID, and organization ID. Optionally, you can configure the `apiUsageByAppFilter` parameter to
+further filter the results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">apiUsageByAppFilter</td>
+<td valign="top"><a href="#apiusagebyappfilter">APIUsageByAppFilter</a></td>
+<td>
+
+API usage by app filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getTopAPIsByAlertCount</strong></td>
+<td valign="top"><a href="#topapisbyalertcount">TopAPIsByAlertCount</a></td>
+<td>
+
+Returns top APIs by the alert count within the specified time range for the given combination of tenant ID,
+environment ID, and organization ID. Optionally, you can configure the `topAPIsByAlertCountFilter` parameter to
+further filter the results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">topAPIsByAlertCountFilter</td>
+<td valign="top"><a href="#topapisbyalertcountfilter">TopAPIsByAlertCountFilter</a></td>
+<td>
+
+Top APIs by alert count filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getAlertSummary</strong></td>
+<td valign="top"><a href="#alertsummaries">AlertSummaries</a></td>
+<td>
+
+Returns a summary of each alert for within the specified time range for the given combination of tenant ID,
```

Comment:
Let's replace "Returns a summary of each alert for within the specified time range" with "Returns a summary for each alert within the specified time range.."?